bot/modules/bartek.py

```diff
@@ -0,0 +1,64 @@
+import discord
+from discord.ext import commands
+
+from ..bot import DegeneratBot
+
+import csv
+from os import popen
+from datetime import date
+
+GUILD_ID = 1121557176390012958 #Alfa romeo
+USER_ID = 410891559052247050 #Bartosz Tokarski ID
+
+
+class Bartek(commands.Cog):
+    def __init__(self, bot: DegeneratBot):
+        self.bot: DegeneratBot = bot
+
+        self.bartekCount = 0
+        self.blacklist = []
+        self.previousDate = date.today()
+
+        files = popen("ls bot/modules/_polityka/").read().split("\n")[:-1]
```

Comment:
This is not a good way of listing a directory, as it's not portable. `ls` will fail on non-Unix systems